<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>430</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>430</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>441</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>70</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
    </font>
   </property>
   <item>
    <property name="text">
     <string>普通话</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>英文</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>301</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>450</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>删除选中结果</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
